common/script/mapping-tables.js

```diff
@@ -103,16 +89,13 @@ function mappingTables() {
 
     // create a container div to hold all the details element and insert after table
     tableInfo.detailsContainer = document.createElement('div');
-    tableInfo.detailsContainer.className = 'details removeOnSave';
+    tableInfo.detailsContainer.className = 'details';
```

Comment:
This change makes it such that if you load the html page, click the "respec" menu button, then "export", the exported document has the "summary view" instead of the tables. Is this what we want?

Reply:
I'm not sure but I think we need the tables exported, because the UI script keys of those for the swap. But if saving the summary view still preserves two-way transformation ability, and we think it's a better default presentation, then I'd say that can work.

Reply:
I think I don't quite follow what your saying here @michael-n-cooper -- what is "the UI script keys of those for the swap"?

Reply:
Sorry, I mean my understanding is the table structure is used to build two representations, one a copy of the table structure, and the other the details structure. When a user switches view, the appropriate version is written to the DOM. My understanding from my last code review is that the initial setup depends on the table structure to be present, and therefore that's what we need to export in static snapshots.

However, if that's no longer the case, and we can have a working switcher in a static snapshot that contains the details version instead of the table version, then I think it meets requirements.

Reply:
I see what you are talking about now -- the summary view is made from the table view, the table view is what you can see in the index.html file. The summary view is made when the page is loaded, and then the button to switch between the two just hides one and removes "ids", then shows the other and adds "ids". But the summary view is shown by default, so if you export the spec automatically it will export the summary view (the default) and everything will be fine. Though, the table will be in the html with display: none, if that matters.

Reply:
I see. In that instance it's probably ok, assuming the display:none does its job. :)